src/main/resources/org/jetbrains/research/tasktracker/ui/main/panel/template/error.html

```diff
@@ -1,10 +1,4 @@
-<!DOCTYPE html>
-<html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <title>Title</title>
-</head>
-<body>
 <h1>Something went wrong</h1>
-</body>
-</html>
\ No newline at end of file
+<h1>Error code : %s</h1>
```

Comment:
Please, wrap code into a container too, add classes for html elements to make it easier to apply different styles to different elements